source/configure/email-templates.rst

```diff
@@ -3,14 +3,14 @@ Email Templates
 
 Mattermost has a few email templates that are sent out when a specific event occurs.
 Most of the time these templates do not need to be modified.
-In case additional modifications are necessary, all available props in each email are listed below.
+In case of additional modifications are necessary, all available props in each email are listed below.
```

Comment:
This change doesn't seem right to me. Please revert this.

Reply:
removed. Thanks @haardikdharma10